react/src/base/inputs/Select.stories.js

```diff
@@ -49,9 +59,15 @@ export const selectBox = () => (
           ],
         },
       ]}
-      name="name"
-      variant="select"
-      label="Select Box Label"
+      name="select"
+    />
+    <SprkIcon
+      iconName="chevron-down"
+      aria-hidden="true"
+      additionalClasses="
+              sprk-c-Icon--filled-current-color
+              sprk-c-Icon--stroke-current-color
+              sprk-b-SelectContainer__icon"
```

Comment:
What's up with these wild spaces? I'm assuming that was prettier.

Reply:
LOL must of been

Reply:
"prettier"

Reply:
i fixed it, looks like it must of been me lol .....